### DISCUSSION TITLE
Why ComfyUI has no release management (release versions)?

### DISCUSSION THREAD

**Discussion by Maslino** [comfyanonymous/ComfyUI#3595] [General]
Why ComfyUI has no release management (release versions)?

**Comment by blepping** [comment]
it adds significant overhead to development having to organize and maintain versions compared to just making changes in an ad hoc way. obviously releases are easier to deal with from a user perspective though.

**Comment by mcmonkey4eva** [comment]
Versioning is actually planned for the upcoming future!
